functests/utils/utils.go

```diff
@@ -19,7 +19,7 @@ func BeforeAll(fn func()) {
 }
 
 func ExecAndLogCommand(name string, arg ...string) ([]byte, error) {
-	out, err := exec.Command(name, arg...).CombinedOutput()
+	out, err := exec.Command(name, arg...).Output()
```

Comment:
Is there a way to at least log the stderr output? Or start the command in such a way that stderr goes to system stderr?

Reply:
on it already. Quoting go docs (https://golang.org/pkg/os/exec/#Cmd.Output):
```
Any returned error will usually be of type *ExitError. If c.Stderr was nil, Output populates ExitError.Stderr. 
```
So the data is there, it's just about properly logging it if `err != nil`.

Reply:
Hmm we might get it indeed via err at least. The docs say: If c.Stderr was nil, Output populates ExitError.Stderr.

Reply:
How will this play our with tests running:  `Expect(err).ToNot(HaveOccurred())`  right after this command ?

Reply:
@fromanirh you can assign a buffer to cmd.Stderr and eventually dump it if the execution returns an error.
See https://stackoverflow.com/questions/31095699/return-cmd-stdout-and-stderr-as-string-instead-of-printing-to-console-in-golang for example

Reply:
@yanirq the function signature will not change